config/cops.yml

```diff
@@ -1,24 +1,26 @@
 Metrics/AbcSize:
-  violation_points: 100_000
+  base_points: 1_000_000
+  violation_points: 70_000
 Metrics/BlockNesting:
-  base_points: 30_000
+  base_points: 100_000
 Metrics/ClassLength:
-  base_points: 400_000
-Metrics/CyclomaticComplexity:
-  base_points: 75_000
-  violation_points: 10_000
+  base_points: 5_000_000
+  violation_points: 35_000
+Metrics/CyclomaticComplexity: # This check is per method
+  base_points: 1_000_000
+  violation_points: 70_000
 Metrics/LineLength:
-  base_points: 20_000
-  violation_points: 5_000
-Metrics/MethodLength:
-  base_points: 50_000
+  base_points: 100_000
```

Comment:
So for a style thing (like missing semicolon in JS) we do 50k. I think that applies here as well. I'm not sure anything would ever be less than 50k, since the 50k number is intended to account for the time it takes to open the file, make the change, save, commit, push... it's about 5 mins even for something small.

For a long line, it doesn't seem to get harder to fix as the line gets longer. Therefore, I'd just do a flat 50k.